crates/nu-protocol/src/config/mod.rs

```diff
@@ -89,6 +89,7 @@ pub struct Config {
     pub error_style: ErrorStyle,
     pub use_kitty_protocol: bool,
     pub highlight_resolved_externals: bool,
+    pub completion_use_ls_colors: bool,
```

Comment:
It would be nice to put this up above with all the other completion config points if possible.